.github/workflows/release-process.yaml

```diff
@@ -0,0 +1,127 @@
+              repo: context.repo.repo,
+              pull_number: context.payload.pull_request.number
+            });
+
+            const approved = reviews.some(review => review.state === 'APPROVED');
+            console.log(`PR has been ${approved ? 'approved' : 'not approved'}`);
+            return approved;
+      - name: Checkout
+        if: steps.checkbox.outputs.result == 'true' && steps.up_to_date.outputs.result == 'true' && steps.pr_approval.outputs.result == 'true' && !github.event.pull_request.draft
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.RENOVATE_GITHUB_TOKEN }}
+      - name: Set up Git and import GPG key
+        if: steps.checkbox.outputs.result == 'true' && steps.up_to_date.outputs.result == 'true' && steps.pr_approval.outputs.result == 'true' && !github.event.pull_request.draft
+        env:
+          GPG_PRIVATE_KEY: ${{ secrets.RENOVATE_PRIVATE_KEY }}
+        run: |
+          echo "${GPG_PRIVATE_KEY}" | gpg --import
+          git config user.name "CCBC Service Account"
+          git config user.email "116113628+ccbc-service-account@users.noreply.github.com"
+          git config user.signingkey "$(gpg --list-secret-keys --with-colons | awk -F: '/sec:/ {print $5}')"
+          git config commit.gpgsign true
+      - name: dev env setup
+        if: steps.checkbox.outputs.result == 'true' && steps.up_to_date.outputs.result == 'true' && steps.pr_approval.outputs.result == 'true' && !github.event.pull_request.draft
+        uses: ./.github/actions/dev-env-setup
+      - name: Setup Sqitch User
+        if: steps.checkbox.outputs.result == 'true' && steps.up_to_date.outputs.result == 'true' && steps.pr_approval.outputs.result == 'true' && !github.event.pull_request.draft
+        run: |
+          sqitch config --user user.name 'CCBC Service Account'
+          sqitch config --user user.email 'ccbc@button.is'
+      - name: Make Release
+        if: steps.checkbox.outputs.result == 'true' && steps.up_to_date.outputs.result == 'true' && steps.pr_approval.outputs.result == 'true' && !github.event.pull_request.draft
+        run: |
+          yarn
+          git checkout "${GITHUB_HEAD_REF}"
+          yarn run release-it --ci --branch="${GITHUB_HEAD_REF}" --git.commitArgs=-n
+      - name: Uncheck the checkbox
+        if: steps.checkbox.outputs.result == 'true' && steps.up_to_date.outputs.result == 'true' && steps.pr_approval.outputs.result == 'true' && !github.event.pull_request.draft
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const checkboxText = "[x] Check to trigger automatic release process";
+            const prNumber = context.payload.pull_request.number;
+            const currentPR = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: prNumber
```

Comment:
Seeing as we're unchecking after creating the release, what would happen if something else is merged? I'm guessing we'll get a conflict so we'd have to resolve this manually. Maybe add some documentation (in the other PR) on what to do in this case for new folks and so all current devs run it the same way.

Reply:
The release process will fail as release-it is smart enough to not allow two releases as it checks the existing tags before. Added info on the two exceptions that can occur on the docs PR :)